### PR TITLE
Swagger redesign/update colors

### PR DESCRIPTION
**Pull request by taraokelly**
#### Abstraction of colors in `_variables.scss`. 
Used this [app](http://chir.ag/projects/name-that-color/#6195ED) to generate unique color names for most colors in this file and defined them as variables (excluding theme colors). 
#### Updated operation colors to follow new UX
`$_color-post`, `$_color-get`, ..., `$_color-options`
#### Deleted `colors.scss`
Method appeared to be unused and variables already defined in `_variables.scss`. Please flag if there is reason for file presence.

(Let me know if this size of PR is small enough or if we can squeeze a little more into them.)

### TBD
- Update theme colors, e.g. `$color-primary`
- Rename color variables e.g.
```scss
// Base Colours

$color-black: #000;
$color-white: #fff;
$color-gray-50: lighten($color-black, 92%); //ebebeb
...
```

...or a similar naming convention.
- Look into moving more specific color vars into their specific block selectors, e.g. $btn-background-color into the .btn class
- Abstract new colors defined in css outside of `_variables.scss` to variables in `_variables.scss`
- Move css defined as inline css in `.js/.jsx` files, and abstract any new colors or change to use existing color variables
- Update any relevant colors in open operations
- Update model colors

### Motivation and Context
Facelift

### How Has This Been Tested?
Manually

### Screenshots
Screen Shot 2019-05-02 at 10 37 36

Screen Shot 2019-05-02 at 10 38 26

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [x] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by shockey** [comment]
> Deleted colors.scss

cc https://github.com/swagger-api/swagger-ui/pull/4973, looks like this PR would supersede it

File: src/style/_variables.scss [hunk 1]
```diff
@@ -1,38 +1,55 @@
...
+$alto-gray: #d9d9d9;
+$mercury-gray: #e4e4e4;
+$concrete-gray: #e8e8e8;
+$alabaster: #f7f7f7;
+$apple-green: #62a03f;
+$green-haze: #009d77;
+$japanese-laurel: #008000;
+$persian-green: #00a0a7;
+$geyser-blue: #d8dde7;
+$doger-blue: #1391ff;
```
Comment (shockey) [inline src/style/_variables.scss]: Did you mean [Dodger blue](https://en.wikipedia.org/wiki/Dodger_blue)? 😉 

image

**Comment by shockey** [comment]
@taraokelly: take a look at my notes in https://github.com/swagger-api/swagger-ui/issues/4460. No need to address it here, but being aware of the context should be helpful 😄 

> Move css defined as inline css in .js/.jsx files, and abstract any new colors or change to use existing color variables

This would be great.

**Comment by shockey** [comment]
please build

File: src/style/_variables.scss [hunk 2]
```diff
@@ -1,38 +1,55 @@
...
-$_color-patch: #50e3c2 !default;
-$_color-disabled: #ebebeb !default;
-$_color-options: #0d5aa7 !default;
+$color-primary-hover: lighten($color-primary, .5%) !default;
 
-$color-green: #008000 !default;
-
-$color-primary-hover: #81b10c !default;
+$_color-post: $green-haze !default;
+$_color-get: $doger-blue !default;
```
Comment (shockey) [inline src/style/_variables.scss]: This is causing a build failure now:

```
Module build failed: 
$_color-get: $doger-blue !default;
            ^
      Undefined variable: "$doger-blue".
      in /var/lib/jenkins/workspace/oss-swagger-ui-pr/src/style/_variables.scss (line 46, column 14)
```

Comment (taraokelly) [reply src/style/_variables.scss]: Whoops, consequence of a rushed changed. Updated 👍

**Comment by shockey** [comment]
thanks @taraokelly! 🎉

**Pull request merged**